docs/calculations.rst

```diff
@@ -811,6 +811,98 @@ The output of the last two lines is::
++------------------------------------------------+-----------------------+--------------------------------------------------------------------------+-----------------------------------------------+
+| linear response iterative coupling             | ``"linear_response"`` | iterative coupling to the solvent via a CIS-like coupling density,       | :cite:`Lunkenheimer2013`, :cite:`Marefat2018` |
+|                                                |                       | the additional term is added to the ADC matrix                           |                                               |
++------------------------------------------------+-----------------------+--------------------------------------------------------------------------+-----------------------------------------------+
+
+The schemes can be selected as described for :ref:`polarisable_embedding`. Note that 
+``environment=True`` does not work for PCM, because the ``"ptss"`` correction is not implemented.
+
+The following example computes the PCM-ADC(2) excited states of para-nitroaniline in water with the
+perturbative linear response scheme.
+
+.. code-block:: python
+
+   import adcc
+   import psi4
+
+   # Run a PCM HF calculation with Psi4
+   mol = psi4.geometry("""
+      C          8.64800        1.07500       -1.71100
+      C          9.48200        0.43000       -0.80800
+      C          9.39600        0.75000        0.53800
+      C          8.48200        1.71200        0.99500
+      C          7.65300        2.34500        0.05500
+      C          7.73200        2.03100       -1.29200
+      H         10.18300       -0.30900       -1.16400
+      H         10.04400        0.25200        1.24700
+      H          6.94200        3.08900        0.38900
+      H          7.09700        2.51500       -2.01800
+      N          8.40100        2.02500        2.32500
+      N          8.73400        0.74100       -3.12900
+      O          7.98000        1.33100       -3.90100
+      O          9.55600       -0.11000       -3.46600
+      H          7.74900        2.71100        2.65200
+      H          8.99100        1.57500        2.99500
+      symmetry c1
+      """)
+
+   psi4.set_options({
+      'basis': "sto-3g",
+      'scf_type': 'pk',
+      'e_convergence': 1e-10,
+      'd_convergence': 1e-10,
+      'pcm': True,
+      'pcm_scf_type': "total"
+   })
+   psi4.pcm_helper("""
+      Units = AU
+      Cavity {
+         Type = GePol
+      }
+      Medium {
+         SolverType = IEFPCM
+         Solvent = Water
+         Nonequilibrium = True
+      }
+   """)
+
+   psi4.core.set_num_threads(4)
+
+   scf_e, wfn = psi4.energy('scf', return_wfn=True)
+
+   # Run a ADC2 calculation with ptLR
+   state = adcc.adc2(wfn, n_singlets=5, conv_tol=1e-8,
+                     environment="ptlr")
+   print(state.describe())
+
+
+
```

Comment:
You should strongly emphasise (even with a warning box) that one needs to adjust the dielectric constant to the optical value for pyscf.

Reply:
Yes your right. Then it would be probably the best to also show this in the example.
Fine with having two examples here?
Think Psi4 would be good to have for the rather complicated PCM input there

Reply:
No problem with more examples I guess 👍

Reply:
Yeah why not, but than output from one is probably sufficient.

Reply:
or not ... it's just a small box, so let's leave it as is.